dmaws/rds.py

```diff
@@ -295,20 +287,27 @@ def clean_database(self):
                     WHERE (
                         SELECT COUNT(*) FROM supplier_frameworks WHERE supplier_id=suppliers.supplier_id
                     ) = 0
+                    AND supplier_id NOT IN (
+                        SELECT DISTINCT supplier_id FROM services
+                    )
                 ), d1 AS (
                     DELETE FROM contact_information WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
                 ), d2 AS (
                     DELETE FROM users WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
                 )
-             DELETE FROM suppliers WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
+            DELETE FROM suppliers WHERE supplier_id IN (SELECT supplier_id FROM dangling_suppliers)
         """)
 
+        # Remove audit events because we don't use them
```

Comment:
Some audit events are used... Admin uses service update events to show diffs (upcoming work on this v.soon from submissions). And some others (draft service updates maybe?) Used to find dates to display.

I'm not saying we should keep these necessarily, but it's wrong to say that we don't use them at all.